examples/fem_system/fem_system_ex3/elasticity_system.C

```diff
@@ -179,20 +184,26 @@ bool ElasticitySystem::element_time_derivative(bool request_jacobian,
   const std::vector<std::vector<Real>> & phi = u_elem_fe->get_phi();
   const std::vector<std::vector<RealGradient>> & grad_phi = u_elem_fe->get_dphi();
 
-  // We set _w_var=_v_var etc in lower dimensions so this is sane:
```

Comment:
Let's keep this comment around here and in side_time_derivative?  It's not a priori obvious that using a variable labeled "the index of w" in a 2D system with only "u" and "v" isn't a disaster waiting to happen.